### DISCUSSION TITLE
Buffering problem

### DISCUSSION THREAD

**Discussion by Luca-Gaetani** [rayanfer32/TgindexPro#12] [Show and tell]
Hi everyone, I have another issue.
I state that I have a Gigabit connection, so I think that the problem is not attributable to the connection.
I don't understand why there is buffering when I play my videos, I also notice that the buffer appears every time at the same point.
I hope someone can help me.

**Comment by rayanfer32** [comment]
Which player are you using?

**Comment by rayanfer32** [comment]
Try incresing the cache value to 10 secs or more

**Comment by Luca-Gaetani** [comment]
I have a problem with VLC and with Emby player (because I'm using telegram to store my videos, sharing in local network the m3u8 playlist) how can I increase the cache value?

**Comment by rayanfer32** [comment]
In vlc you can search for cache in preferences

**Comment by rayanfer32** [comment]
Chsnge the vlc settings and save.. And restart vlc

**Comment by Luca-Gaetani** [comment]
I changed the setting on vlc, the problem persists but it happens more rarely, any idea?

**Comment by rayanfer32** [comment]
Are you hosting the bot locally or in the repl?

**Comment by Luca-Gaetani** [comment]
I'm hosting the bot locally

**Comment by rayanfer32** [comment]
Check if cryptg is installed

**Comment by Luca-Gaetani** [comment]
Yes cryptg is installed

**Comment by rayanfer32** [comment]
Whats the resolution of the video  you are  trying to play.

**Comment by rayanfer32** [comment]
4k won't work

**Comment by rayanfer32** [comment]
Currently i have had  no issues with upto 1080p

**Comment by Luca-Gaetani** [comment]
it's a 1080p video

**Comment by rayanfer32** [comment]
I have no clue what's causing the buffering... You can try deploying it on repl.it or try to play different video

**Reply by Luca-Gaetani** [reply]
I'll try and let you know

**Comment by Luca-Gaetani** [comment]
Could the weight of the single file be the cause of the problem?
Let me explain, I have several videos that weigh more than 2gb and for this reason I have divided them in order to obtain parts of 2gb.
Now I realized that this problem only occurs with the parts of those videos that weigh more, could this be the cause?

**Reply by Luca-Gaetani** [reply]
So a problem of video bitrate

**Comment by rayanfer32** [comment]
Oh well... Your video has high Bitrate

**Reply by Luca-Gaetani** [reply]
Exactly
